docs/user/docs/tokens.md

```diff
@@ -0,0 +1,55 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Token Authentication
+
+To authenticate against the API you can either use a Bearer token (which the
+browser usually uses) or authenticate with a longer lived basic authentication
+token which can be used e.g. in scripts.
+
+## Token Creation
```

Comment:
```suggestion
## PAT Creation
```
Let's call it PAT. Otherwise it's not clear whether we are talking about basic authentication tokens or Bearer tokens.